blog/content/post/2023-02-24-msi_ms7d25_v1.1.1_release.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Dasharo compatible with MSI PRO Z690-A release v1.1.1
+abstract: 'A short overview of changes introduced in v1.1.1
+           release of Dasharo compatible with MSI PRO Z690-A
+	   with a short demo of Qubes OS 4.1.2.'
+cover: /covers/image-file.png
+author: michal.zygowski
+layout: post
+published: true
+date: 2023-02-24
+archives: "2023"
+
+tags:
+  - firmware
+  - coreboot
+  - Dasharo
+categories:
+  - Firmware
+
+
+---
+
+# Introduction
+
+New year and new exciting news. Dasharo Team has just released a new version of
+Dasharo firmware compatible with MSI PRO Z690-A. Last v1.1.0 version was
+released on 22th of November 2022 and after 3 months the time has come for v1.1.1.
+Let's see what's new.
```

Comment:
Below blog post looks massive and we just change patch level, if I would knew so many changes are introduced I would call this release v1.2.0 - maybe at least state that we squeezed a lot in this release.

Reply:
I have been working on some issues and features in free time, that is why there is much more than anticipated. I don't think it needs v1.2.0. Last time we bumped minor version because of coreboot codebase change which is significant. I thought fixes based on the same coreboot base should only bump the patch version. I guess we have to standardize it if we have different opinions here.

Reply:
My logic:

- we update coreboot base -> this is significant change -> bump minor

- we fix bugs on the existing coreboot base -> these are just bug fixes and improvements -> do not qualify for higher minor version, simply bump the patch version

Reply:
@miczyg1 please explain that to readers of blog post. TBH if we agree to such methodology we should have it in dev process documentation. @macpijan cc

Reply:
Why? You brought up the versioning but readers will rather be more interested what changed not why it is v1.1.0 instead of v1.2.0

Reply:
@miczyg1 I'm reader and I'm interested why massive change has just minor number update.

Reply:
https://github.com/3mdeb/news-and-ideas/pull/513/commits/954e457ffcebf59ba34be75e772c879aada8cac3